gui/src/common/const-string.json

```diff
@@ -17,5 +17,22 @@
   "throughput": "throughput",
   "latency": "latency",
   "iops": "iops",
-  "alert": "alert"
+  "alert": "alert",
+  "features": {
```

Comment:
We can have separate ts file for these constants like we have for permissions. Lets discuss on this.

Reply:
Added seperate .ts file for these constants.